src/pages/Auth/Login.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState } from "react";
+import "./Auth.css";
+import VisibilityOutlinedIcon from "@mui/icons-material/VisibilityOutlined";
+import VisibilityOffOutlinedIcon from "@mui/icons-material/VisibilityOffOutlined";
+import { Link } from "react-router-dom";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [eye, setEye] = useState(true);
+  const emailRef = useRef();
+  const passwordRef = useRef();
+
+  const changeHandler = (e) => {
+    switch (e.target.name) {
+      case "email":
+        setEmail(() => {
+          let newValue = e.target.value;
+          newValue
+            ? emailRef.current.classList.add("field-label-hover")
```

Comment:
DOM manipulations are a strict no in React. Replace this with controlled components.